### ISSUE TITLE
Add a Github Action for ABI/API Compliance

### ISSUE THREAD

**Issue by kylophone** [Netflix/vmaf#590]
We'll need this for `libvmaf_rc` only, since `libvmaf` is nearly deprecated.

https://lvc.github.io/abi-compliance-checker/

**Comment by 1480c1** [comment]
Are you looking for abi/api compliance between a pr and master?

**Comment by kylophone** [comment]
Yes, exactly. I've seen this done elsewhere, but I've never actually tried these tools myself.
